README.md

```diff
@@ -33,6 +33,21 @@ pnpm dev
 bun dev
 ```
 
+Choose environment using the PROFILE environment variable, such as:
+
+```bash
+export PROFILE=testnet
+```
+
+the value has to correspond to one of the `.env.` file endings
+
+> [!NOTE]
+> ‼️ **<span style="color:red;">Warning: DO NOT USE IN PRODUCTION!</span>**
+>
+> To avoid **CORS errors** when running against testnet from localhost we can proxy the calls to the RIF Wallet Services via a local loop that strips the CORS headers on the way out and adds the expected response ones on the way in.
```

Comment:
To avoid a cors issue you should be running the server locally. I'm not going to fight it now, but this workaround creates bloat.

Reply:
> I'm not going to fight it now
Thank you, let's focus on the deployment now.
I'm creating a document to keep track of all the things we need to agree on